challenges/0-hello-world/php/lasatd/helloworld.php

```diff
@@ -0,0 +1,5 @@
+<?php
+
+    echo "Hello World";
```

Comment:
Favor arrumar a identação da linha
```suggestion
echo "Hello World";
```